test/built-ins/Temporal/now/timeZone/extensible.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.now.timezone
+description: Temporal.now.timeZone is extensible.
+info: |
+  ## 17 ECMAScript Standard Built-in Objects
+
+  Unless specified otherwise, the [[Extensible]] internal slot
+  of a built-in object initially has the value true.
+features: [Temporal]
+---*/
+
+assert(Object.isExtensible(Temporal.now.timeZone));
```

Comment:
```suggestion
assert.sameValue(Object.isExtensible(Temporal.now.timeZone), true);
```

Reply:
This looks equivalent to me. Could you say a little about why we ought to use `assert.sameValue` instead of `assert`?

Reply:
I understood the former to be preferred in test262, but I may be misremembering. Happy to go either way.

Reply:
@jugglinmike you're right, this is fine as is. 

@Ms2ger `assert()` is discouraged in cases such as `assert(foo === "something interesting")`, which is better written as `assert.sameValue(foo, "something interesting")` because you'll get a more meaningful error message.

Reply:
Sounds like I misremembered, thanks for the clarification. I couldn't find this discouragement in the documentation at https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#test-environment - could you add it?